showcase/legends/searchable-discrete-color.js

```diff
@@ -45,24 +48,35 @@ export default class Example extends React.Component {
   _clickHandler(item) {
     const {items} = this.state;
     item.disabled = !item.disabled;
-    this.setState({items});
+    this.setState({
+      items
+    });
   }
 
   _searchChangeHandler(searchText) {
     this.setState({searchText});
   }
 
   render() {
-    const {items, searchText} = this.state;
+    const {items, searchText, useDefaultInputClass} = this.state;
+    const content = useDefaultInputClass ? 'REMOVE DEFAULT INPUT CLASS' : 'USE DEFAULT INPUT CLASS';
     return (
-      <SearchableDiscreteColorLegend
-        height={200}
-        width={300}
-        onSearchChange={this._searchChangeHandler}
-        searchText={searchText}
-        onItemClick={this._clickHandler}
-        items={items}
-      />
+      <div>
```

Comment:
I don't know if you need to go to the trouble of adding class toggling here, i think a more effective testing method would be to use an example inputClassName here, and then write a unit test against it.

Reply:
Ok that makes sense.  I wasn't sure if this was a property that needed to be demonstrated in the show case or not

Reply:
generally, i only add buttons for properties that are visually impacted